include/alp/reference/blas3.hpp

```diff
@@ -1649,84 +1577,10 @@ namespace alp {
 			return SUCCESS;
 		}
 
-		RC rc = internal::set_band< 0 >( C, A );
-
-		if( rc == SUCCESS ) {
-			internal::setInitialized( C, true );
-		}
-
-		return rc;
+		internal::setInitialized( C, true );
```

Comment:
Shouldn't this depend on the init status of the A?

Reply:
Initi status of the A is checked in line 1575 and the function returns if it is not. If we arrive to this point, it means that A is initialized.